src/proteomes/components/BuscoAbbr.tsx

```diff
@@ -0,0 +1,3 @@
+export default () => (
```

Comment:
As we will have `abbr` for a lot more field names, it might be worth making a map somewhere in shared utils?

Reply:
Good idea - updated.